### PR TITLE
Add dynatrace-oneagent-operator 0.6.0

### PR DESCRIPTION
**Pull request by DTMad**

### CODE REVIEW

**Comment by DTMad** [comment]
Hey @guangbochen !
Any update on the review?
Can I help you somehow? :)

Regards,
Marco

**Comment by guangbochen** [comment]
Hi @DTMad since this PR is a helm_v3 chart so if you need to keep the v3 style please submit the PR against the https://github.com/rancher/helm3-charts repo, and helm v3 will be supported in rancher 2.4.x.

for current rancher version which is Rancher 2.3.x, we only support helm v2, so if you like to publish the dynatrace chart please roll back to the v2 format, thanks

**Comment by guangbochen** [comment]
and sorry for the late reply as I though our partner manager has already send out the update to you guys, thanks.

**Comment by DTMad** [comment]
Hey @guangbochen !

Thanks for the info :)
I tried to run our chart on Rancher before (with helm2) and it worked since we added a separate step in the 'app-readme.md' where we inform the user to add the CustomResourceDefinition in before.
Shouldn't this do the trick for now? :D

From which version on will these Helm 3 charts be available in Rancher? 

Thanks,
Marco

File: charts/dynatrace-oneagent-operator/v0.6.0/crds/customresourcedefinition.yaml [hunk 1]
```diff
@@ -0,0 +1,368 @@
...
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
```
Comment (guangbochen) [inline charts/dynatrace-oneagent-operator/v0.6.0/crds/customresourcedefinition.yaml]: you can move this CRD file to the templates folder and use `crd-install` hook to pre-install the CRDs so user doesn't need to apply this file manually.
``` 
annotations:
    "helm.sh/hook": "crd-install"
```

**Comment by DTMad** [comment]
Hey @guangbochen !
We decided to close this review and create a new one for the helm3 repo:
https://github.com/rancher/helm3-charts/pull/1

Could you be so kind and review it?

Thanks,
Marco

**Pull request closed without merging**